qa/tasks/vstart_runner.py

```diff
@@ -423,7 +423,7 @@ def _do_run(self, args, check_status=True, wait=True, stdout=None,
 
         # Filter out helper tools that don't exist in a vstart environment
         args = [a for a in args if a not in ('adjust-ulimits',
-                                             'ceph-coverage')]
+                                             'ceph-coverage', 'None/archive/coverage')]
```

Comment:
I think it's from here:

https://github.com/ceph/ceph/blob/master/qa/tasks/ceph_manager.py#L1725

Maybe we should filter out all the `*/archive/coverage` ?

Reply:
Not required. It's always set to `None` in vstart_runner https://github.com/ceph/ceph/pull/44305/files/d01ef7682c1b1a9a6de28d458d64832708b0b3e2#R797

Reply:
Okay.